packages/convergence/package.json

```diff
@@ -9,7 +9,8 @@
   "scripts": {
     "build": "rollup --config",
     "test": "mocha --opts ./tests/mocha.opts ./tests",
-    "postpublish": "../../bin/tag-pkg"
+    "postpublish": "../../bin/tag-pkg",
+    "lint": "./node_modules/.bin/eslint ./src/**.js ./tests/**.js"
```

Comment:
We can remove the `./node_modules/.bin/` bit because when we run `yarn <command>`, yarn will look in that directory first for binaries.

We should maybe lint the whole package too, so `eslint ./` is probably what we want.